QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -72,28 +42,33 @@ public void event() {
     private static FloatInput winchSpeed = ControlInterface.mainTuning.getFloat("Elevator Winch Speed +M", 1.0f);
 
     public static void setup() {
-
-        Ticker updateCAN = new Ticker(100);
-        Cluck.publish("Elevator CAN Enable", winchCAN.asEnable());
-        Cluck.publish("Elevator CAN Bus Voltage", FloatMixing.createDispatch(winchCAN.asStatus(ExtendedMotor.StatusType.BUS_VOLTAGE), updateCAN));
-        Cluck.publish("Elevator CAN Output Current", FloatMixing.createDispatch(winchCAN.asStatus(ExtendedMotor.StatusType.OUTPUT_CURRENT), updateCAN));
-        Cluck.publish("Elevator CAN Output Voltage", FloatMixing.createDispatch(winchCAN.asStatus(ExtendedMotor.StatusType.OUTPUT_VOLTAGE), updateCAN));
-        Cluck.publish("Elevator CAN Temperature", FloatMixing.createDispatch(winchCAN.asStatus(ExtendedMotor.StatusType.TEMPERATURE), updateCAN));
-        Cluck.publish("Elevator CAN Any Fault", QuasarHelios.publishFault("elevator-can", winchCAN.getDiagnosticChannel(ExtendedMotor.DiagnosticType.ANY_FAULT)));
-        Cluck.publish("Elevator CAN Bus Voltage Fault", BooleanMixing.createDispatch(winchCAN.getDiagnosticChannel(ExtendedMotor.DiagnosticType.BUS_VOLTAGE_FAULT), updateCAN));
-        Cluck.publish("Elevator CAN Temperature Fault", BooleanMixing.createDispatch(winchCAN.getDiagnosticChannel(ExtendedMotor.DiagnosticType.TEMPERATURE_FAULT), updateCAN));
```

Comment:
Sorry - I left out a crucial file from the commit. It takes care of this in an abstracted way.